app/controllers/fe/concerns/answer_pages_controller_concern.rb

```diff
@@ -33,8 +33,8 @@ def update
     questions = @presenter.all_questions_for_page(params[:id])
     questions.set_filter(get_filter)
     questions.post(answer_params, @answer_sheet)
-
     questions.save
+    Fe::UpdateReferenceSheetVisibilityJob.perform_later(@answer_sheet, questions.questions.collect(&:id))
```

Comment:
I was going to do that, but then I read this: http://edgeguides.rubyonrails.org/active_job_basics.html#globalid

> Active Job supports GlobalID for parameters. This makes it possible to pass live Active Record objects to your job instead of class/id pairs, which you then have to manually deserialize. Before, jobs would look like this:

Does that fulfill the same thing as an id and find by itself?  I thought from reading those docs it would